source/globalCommands.py

```diff
@@ -3667,8 +3681,8 @@ def script_reportLinkDestination(
 
 	@script(
 		description=_(
-			# Translators: input help mode message for Report URL of navigator link in a window command
-			"Reports the destination URL of the link in the navigator object in a window, "
+			# Translators: input help mode message for Report URL of a link in a window command
+			"shows the destination URL of the link at the position of caret or focus in a window, "

```

Comment:
Displays might be more accurate? Also fixing the capital letter for the start of the sentence

```suggestion
			"Displays the destination URL of the link at the position of caret or focus in a window, "
```

```suggestion
			"Shows the destination URL of the link at the position of caret or focus in a window, "
```